packages/scene-graph-cocos-animation/client/src/SceneGraphAnimation.ts

```diff
@@ -139,40 +140,93 @@ export default class SceneGraphAnimation implements sgmed.SceneExporterPlugin, s
     return animationComponents;
   }
 
-
-  private extendNodeWithAnimationComponent(
+  private extendNodesWithAnimationComponent(
```

Comment:
CCM, ABC Metrics or whatever will tell that this method should be improved.
It should be done in next commit.